README.md

```diff
@@ -91,7 +91,24 @@ To ensure consistency throughout the source code, keep these rules in mind as yo
 * Java files **must be** formatted using [Intellij IDEA's code style](http://confluence.jetbrains.com/display/IntelliJIDEA/Code+Style+and+Formatting).
 * Web apps JavaScript files **must follow** [Google's JavaScript Style Guide](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml).
 * AngularJS files **must follow** [John Papa's Angular 1 style guide] (https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md).
-
+         
+## FAQs      
```

Comment:
I don't think you did a successful rebase. Otherwise this block of code shouldn't be here.